app/services/now-playing.js

```diff
@@ -63,11 +116,28 @@ export default Service.extend({
     slugFromHost = slugFromHost ? slugFromHost : 'wqxr';
     this.set('slugFromHost', slugFromHost);
     this.set('slug', slugFromHost);
+
+    this.poll.addPoll({
```

Comment:
Is this polling needed because WOMS doesn't yet push show metadata? Or is it to keep the playlist history up to date with track changes?

Reply:
This is just a timer that goes off to update the schedule based on the system clock on the client-end of things—no network requests happen on that poll.

I thought I could tie into ember-clock and set up a computed property based on `clock.second` like we've done elsewhere, but something was funky when using that in a service layer—components and controllers it works fine. I wanted to dig in more to understand why, but this gets us to where we want to be.